src/main/java/seedu/duke/atomichabit/feature/AtomicHabit.java

```diff
@@ -0,0 +1,25 @@
+package seedu.duke.atomichabit.feature;
+
+public class AtomicHabit {
+    private String description;
+    private int count;
+
+    public AtomicHabit(String description) {
+        this.description = description;
+        this.count = 1;
```

Comment:
Minor documentation nit - might be good to comment why count is initialized to 1